data/json/items/gun/38.json

```diff
@@ -351,5 +351,137 @@
       }
     ],
     "melee_damage": { "bash": 6 }
+  },
+  {
+    "id": "uberti_1873",
+    "copy-from": "rifle_manual",
+    "looks_like": "modular_ar15",
+    "type": "GUN",
+    "name": { "str": "reproduction .357 lever-action rifle" },
+    "description": "This fine lever-action rifle is chambered in the powerful .357 Magnum cartridge, with an 20\" barrel with a tube fed magazine.  Go get 'em cowboy!",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "uberti_1873",
+        "name": { "str": "Uberti 1873" },
+        "description": "The 1873 is a rifle platform developed all the way in, unsurprisingly, 1873.  This one is a fairly faithful Uberti reproduction, although the original 1873s pre-dated .357 Magnum, and even .38 Special.  This one has a case-hardened finish, wood furniture, and octagonal barrel."
+      }
+    ],
+    "//": "Stats from the .357 Short Rifle https://www.uberti-usa.com/cartridge-rifles/1873-rifle-and-carbine",
+    "weight": "3719 g",
+    "volume": "2718 ml",
+    "longest_side": "1117 cm",
+    "barrel_length": "508 mm",
+    "price": "1429 USD",
+    "price_postapoc": "15 USD",
+    "to_hit": { "grip": "none", "length": "hand", "surface": "line", "balance": "uneven" },
+    "material": [ "steel", "wood" ],
+    "color": "dark_gray",
+    "range": 7,
+    "ammo": [ "357mag", "38" ],
+    "dispersion": 140,
+    "durability": 8,
+    "blackpowder_tolerance": 56,
+    "clip_size": 10,
+    "valid_mod_locations": [
+      [ "grip", 1 ],
+      [ "mechanism", 2 ],
+      [ "sights mount", 1 ],
+      [ "sling", 1 ],
+      [ "stock mount", 1 ],
+      [ "rail mount", 1 ],
+      [ "underbarrel mount", 1 ]
+    ],
+    "flags": [ "NO_TURRET" ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "357mag": 10, "38": 10 } } ],
+    "melee_damage": { "bash": 12 }
+  },
+  {
+    "id": "rossi_r92_357_20",
+    "copy-from": "rifle_manual",
+    "looks_like": "modular_ar15",
+    "type": "GUN",
+    "name": { "str": ".357 lever-action rifle" },
+    "description": "This lever-action rifle is chambered in the powerful .357 Magnum cartridge, with an 20\" barrel with a tube fed magazine.  Go get 'em cowboy!",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "rossi_r92_357_20",
+        "name": { "str": "Rossi R92 20\"", "str_pl": "Rossi R92s 20\"" },
```

Comment:
It may be better to use the double prime symbol `″` to denote inches, since it wouldn't need escaping.

Reply:
Does using the escape have any flaws with it?

Reply:
Using a dedicated (and correct) symbol makes working with the text containing it less error-prone.

Reply:
As far as I can tell, " is as correct as the double prime. I'm unsure how using the double prime would make it less error-prone

Reply:
> Prime marks and quotation marks are not the same thing. <..> a double prime mark is used to show inches or seconds.

— [*Digital Typography: Practical Advice for Getting the Type You Want When You Want It*](https://archive.org/details/digitaltypograph0000gold/page/66)

Reply:
If you want to replace all the inch notations with double prime, be my guest, but I have no intention or want to change it. It's not going to look any different than a double prime in game anyways